DevCycle/Utils/UserPreferences.swift

```diff
@@ -0,0 +1,16 @@
+class UserPreferences {
+    private var defaults: UserDefaults = UserDefaults.standard
+    private var anonymousUserIdKey = "anonymousUserId"
```

Comment:
this can be a `let` since presumably will never change, also check the repo for naming conventions for constants

Reply:
thanks, updated!